src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackJCloudsProvider.java

```diff
@@ -36,6 +36,7 @@
 import javax.ws.rs.core.Response;
 
 import org.jclouds.compute.ComputeService;
+import org.jclouds.ec2.domain.KeyPair;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.jclouds.ec2.domain.KeyPair'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)